src/client/src/ui/analytics/components/Analytics.tsx

```diff
@@ -6,12 +6,19 @@ import AnalyticsBarChart from './AnalyticsBarChart'
 import PositionsBubbleChart from './positions-chart/PositionsBubbleChart'
 
 import { CurrencyPair } from 'rt-types'
-import PNLChart from './pnlChart/PNLChart'
-
-import { PopoutIcon } from 'rt-components'
-import { ThemeProvider } from 'rt-theme'
-import { AnalyticsStyle, BubbleChart, Chart, Controls, Header, LastPosition, PopoutButton, Title } from './styled'
 
+import {
+  HrBar,
+  AnalyticsStyle,
+  BubbleChart,
+  LinearChartStyle,
+  LastPosition,
+  Title,
+  LastPositionWrapper,
+  USDspan,
+} from './styled'
+import AnalyticsHeader from './AnalyticsHeader'
+import ProfitLossLineChart from './AnalyticsLineChart'
```

Comment:
I understand why you've used a different name for the import, but I think potentially this is confusing and not a pattern I'd adopt wholesale.